model/installation.go

```diff
@@ -23,6 +24,13 @@ type Installation struct {
 	DeleteAt       int64
 	LockAcquiredBy *string
 	LockAcquiredAt int64
+	GroupOverrides map[string]string `json:"GroupOverrides,omitempty"`
+
+	// configconfigMergedWithGroup is set when the installation configuration
+	// has been overridden with group configuration. This value can then be
+	// checked later to determine whether the installation is safe to save or
+	// not.
+	configMergedWithGroup bool
```

Comment:
Private fields don't get serialized to JSON do they? If so might be worth adding `json:"-"`

Reply:
Yeah, they don't so we good there.